test/active_model_serializers/test/serializer_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
+require 'rails-controller-testing' if Rails::VERSION::MAJOR >= 5
```

Comment:
@maurogeorge that's kind of hilarious because it was removed from rails due to it being too intimate a test https://github.com/rails/rails/issues/18950
We can remove that test and the dependency.

Do you have a link to the file you're basing this on? Would be nice to see.